client/my-sites/domains/components/form/country-select.jsx

```diff
@@ -57,13 +58,13 @@ const CountrySelect = createReactClass( {
 		} else {
 			options = options.concat( [
 				{ key: 'select-country', label: this.props.translate( 'Select Country' ), value: '' },
-				{ key: 'divider1', label: '', disabled: 'disabled', value: '-' },
+				{ key: 'divider' + dividerCount, label: '', disabled: 'disabled', value: '-' },
 			] );
 
 			options = options.concat(
 				countriesList.map( ( country, index ) => {
 					if ( isEmpty( country.code ) ) {
-						return { key: 'divider2', label: '', disabled: 'disabled', value: '-' };
+						return { key: 'divider' + ++dividerCount, label: '', disabled: 'disabled', value: '-' };
```

Comment:
One alternative way of resolving this is to just use the index as the key.
Thus, for this line, it would be:
```
return { key: index, label: '', disabled: 'disabled', value: '-' }
```

and the up-comping `country-seletct-${ index } -${ country.code }` would also be simplified as:
```
key: index,
```
In this way, we also don't need to introduce `dividerCount` :)

I'm not sure why the original version uses a descriptive key like these. However, I personally think we could consider to keep it minimum as long as it's unique, since it is something react uses internally.

I'm totally fine with your approach, though. This is just a soft comment :)